artist_query/lib/artist_query.ex

```diff
@@ -0,0 +1,41 @@
+defmodule ArtistQuery do
+  def main(_args) do
+    artist = prompt
+    load_songs("music.csv")
+    |> Enum.filter_map(&(String.upcase(&1["Artist"])) == artist, &(&1["Name"]))
+    |> put_songs(titlecase(artist))
+
+    main([]) # Repeat forever
```

Comment:
That will also allow you to give a good name to whatever `main` is currently doing.